### PR TITLE
Fixed  ticket sales, Error in discount total, Attendee's actual ticket price and Orders&Attendee's tab 

### LINKED ISSUE #2878: Errors in calculation of ticket sales
The calculation of ticket sales is erroneous. If discounted or free tickets are sold the sales overview still shows the normal ticket price instead of the real ticket price.

Expected: Show the real sales revenue in the sales column.

The total already shows the correct income.

screenshot from 2017-01-08 23-51-36

E.g. if you have admin access compare: https://eventyay.com/events/6/tickets/

### LINKED ISSUE #2879: Show actually paid ticket price in view Attendees section
The view attendees section of event organizers has a column with the ticket price. Please show the real price paid here (e.g. if people are using discounts) instead of the normal ticket price.

screenshot from 2017-01-09 00-01-33

### LINKED ISSUE #2909: Ticket get discounted even if the ticket not selected for discount code.
screenshot from 2017-01-10 21-57-34
screenshot from 2017-01-10 21-58-08

### LINKED ISSUE #2912: Attendees showing only completed orders
Attendees in ticket tab has options for all different tabs but shows on completed orders only.

### PR DESCRIPTION
**Pull request by magdalenesuo**
Fixes #2909 
Fixes #2878 
Fixes #2879
Fixes #2912

Fix for #2878 
screenshot from 2017-01-11 01-36-23

Fix for #2909 
screenshot from 2017-01-11 01-37-47
screenshot from 2017-01-11 01-37-18

Fix for #2879
screenshot from 2017-01-11 16-51-19
screenshot from 2017-01-11 16-36-47
screenshot from 2017-01-11 17-51-44
 
Fix for #2912
screenshot from 2017-01-11 17-25-42
screenshot from 2017-01-11 17-26-16

### CODE REVIEW

**Comment by SaptakS** [comment]
Looks good to me as of now. Will review completely once the PR is completed.

File: app/templates/gentelella/admin/event/tickets/orders.html [hunk 1]
```diff
@@ -103,6 +106,8 @@
                         <span class="label label-success">{{ order.status | capitalize }}</span>
                     {% elif order.status == 'pending' %}
                         <span class="label label-warning">{{ order.status | capitalize }}</span>
+                    {% elif order.status == 'placed' %}
+                        <span class="label label-warning">{{ order.status | capitalize }}</span>
```
Comment (SaptakS) [inline app/templates/gentelella/admin/event/tickets/orders.html]: change this to `label-info` please

File: app/views/users/ticket_sales.py [hunk 2]
```diff
@@ -102,11 +110,38 @@ def display_orders(event_id):
 
 
 @event_ticket_sales.route('/attendees/')
-def display_attendees(event_id):
+def display_attendees(event_id,):
```
Comment (SaptakS) [inline app/views/users/ticket_sales.py]: remove the extra comma

**Comment by niranjan94** [comment]
@magdalenesuo can be merged ?

**Comment by magdalenesuo** [comment]
@niranjan94 Yes. Can be merged :)

**Pull request merged**